Description: Do not enable dynarec on ppc when built with --enable-dynarec=auto
 PowerPC dynarec code seems unsupported and not working.
Author: Andrey Rahmatullin <wrar@wrar.name>
Forwarded: not-needed
Last-Update: 2012-03-03

--- a/configure.ac
+++ b/configure.ac
@@ -220,8 +220,6 @@ if test "x$DYNARECSEL" = xauto; then
 		DYNARECSEL="x86"
 	elif expr x"$build_cpu" : 'xx86_64' > /dev/null; then
 		DYNARECSEL="x86_64"
-	elif expr x"$build_cpu" : 'xpowerpc' > /dev/null; then
-		DYNARECSEL="ppc"
 	fi
 fi
 
